python/lsst/meas/algorithms/subtractBackground.py

```diff
@@ -36,6 +37,51 @@
 import lsst.pipe.base as pipeBase
 
 
+@contextmanager
+def backgroundFlatContext(maskedImage, doApply, backgroundToPhotometricRatio=None):
+    """Context manager to convert from photometric-flattened to background-
+    flattened image.
+
+    Parameters
+    ----------
+    maskedImage : `lsst.afw.image.MaskedImage`
+        Masked image (image + mask + variance) to convert from a
+        photometrically flat image to an image suitable for background
+        subtraction.
+    doApply : `bool`
+        Apply the conversion? If False, this context manager will not
+        do anything.
+    backgroundToPhotometricRatio : `lsst.afw.image.Image`, optional
+        Image to convert background-flattened image to
+        photometric-flattened image (and back).
```

Comment:
When you fixed the implementation on the later commit, I don't think the docs had to change, because I don't feel like "apply" implies either "multiply" or "divide".  But I also think we want the docs to talk about going from a photometric-flattened image to a backtround-flattened one and back, since that's what the context actually does.  How about:

> Image to multiply a photometrically-flattened image by to obtain a background-flattened image.